staging/src/k8s.io/api/extensions/v1beta1/zz_generated.prerelease-lifecycle.go

```diff
@@ -20,254 +20,326 @@ limitations under the License.
 
 package v1beta1
 
-// Introduced is an autogenerated function, returning the release in which the API struct was introduced as int versions of major and minor for comparison.
+// APILifecycleIntroduced is an autogenerated function, returning the release in which the API struct was introduced as int versions of major and minor for comparison.
 // It is controlled by "k8s:prerelease-lifecycle-gen:introduced" tags in types.go.
-func (in *DaemonSet) Introduced() (int64, int64) {
+func (in *DaemonSet) APILifecycleIntroduced() (major, minor int) {
 	return 1, 1
 }
 
-// Deprecated is an autogenerated function, returning the release in which the API struct was or will be deprecated as int versions of major and minor for comparison.
+// APILifecycleDeprecated is an autogenerated function, returning the release in which the API struct was or will be deprecated as int versions of major and minor for comparison.
 // It is controlled by "k8s:prerelease-lifecycle-gen:deprecated" tags in types.go or  "k8s:prerelease-lifecycle-gen:introduced" plus three minor.
-func (in *DaemonSet) Deprecated() (int64, int64) {
+func (in *DaemonSet) APILifecycleDeprecated() (major, minor int) {
 	return 1, 8
 }
 
-// Removed is an autogenerated function, returning the release in which the API is no longer served as int versions of major and minor for comparison.
+// APILifecycleReplacement is an autogenerated function, returning the group, version, and kind that should be used instead of this deprecated type.
+// It is controlled by "k8s:prerelease-lifecycle-gen:replacement=<group>,<version>,<kind>" tags in types.go.
+func (in *DaemonSet) APILifecycleReplacement() (group, version, kind string) {
```

Comment:
how about schema.GroupVersionKind from apimachinery.  The APIs are guaranteed to have it as a dep because we checked TypeMeta before